tests/dummy/app/pods/area/template.hbs

```diff
@@ -1,4 +1,5 @@
 <div class="dummy-body textarea">
+  <div class="section-header">frost-textarea</div>
```

Comment:
The only reason select has the headers is that there are two sub components: frost-select and frost-multi-select.  I added it to distinguish the two.   I did not add update the other components for the same reason as Steve mentioned above - they are not necessary for the docs.  I am assuming we would keep the select headers.